extensions/mssql/src/objectExplorerNodeProvider/connection.ts

```diff
@@ -6,226 +6,93 @@
+	private _sqlClusterConnObj: sqlops.connection.Connection;
+	private _sqlClusterConnProfile: sqlops.IConnectionProfile;
 	private _host: string;
-	private _knoxPort: string;
-
-	constructor(private connectionInfo: sqlops.ConnectionInfo, private connectionUri?: string, private _connectionId?: string) {
-		if (!this.connectionInfo) {
-			throw new Error(localize('connectionInfoMissing', 'connectionInfo is required'));
-		}
-
-		if (!this._connectionId) {
-			this._connectionId = UUID.generateUuid();
-		}
-	}
-
-	public get uri(): string {
-		return this.connectionUri;
-	}
-
-	public saveUriWithPrefix(prefix: string): string {
-		let uri = `${prefix}${this.host}`;
-		uri = appendIfExists(uri, constants.knoxPortPropName, this.knoxport);
-		uri = appendIfExists(uri, constants.userPropName, this.user);
-		uri = appendIfExists(uri, constants.groupIdPropName, this.connectionInfo.options[constants.groupIdPropName]);
-		this.connectionUri = uri;
-		return this.connectionUri;
-	}
-
-	public async tryConnect(factory?: FileSourceFactory): Promise<sqlops.ConnectionInfoSummary> {
-		let fileSource = this.createHdfsFileSource(factory, {
-			timeout: this.connecttimeout
-		});
-		let summary: sqlops.ConnectionInfoSummary = undefined;
-		try {
-			await fileSource.enumerateFiles(constants.hdfsRootPath);
-			summary = {
-				ownerUri: this.connectionUri,
-				connectionId: this.connectionId,
-				connectionSummary: {
-					serverName: this.host,
-					databaseName: undefined,
-					userName: this.user
-				},
-				errorMessage: undefined,
-				errorNumber: undefined,
-				messages: undefined,
-				serverInfo: this.getEmptyServerInfo()
-			};
-		} catch (error) {
-			summary = {
-				ownerUri: this.connectionUri,
-				connectionId: undefined,
-				connectionSummary: undefined,
-				errorMessage: this.getConnectError(error),
-				errorNumber: undefined,
-				messages: undefined,
-				serverInfo: undefined
-			};
-		}
-		return summary;
-	}
-
-	private getConnectError(error: string | Error): string {
-		let errorMsg = utils.getErrorMessage(error);
-		if (errorMsg.indexOf('ETIMEDOUT') > -1) {
-			errorMsg = LocalizedConstants.msgTimeout;
-		} else if (errorMsg.indexOf('ENOTFOUND') > -1) {
-			errorMsg = LocalizedConstants.msgTimeout;
-		}
-		return localize('connectError', 'Connection failed with error: {0}', errorMsg);
-	}
-
-	private getEmptyServerInfo(): sqlops.ServerInfo {
-		let info: sqlops.ServerInfo = {
-			serverMajorVersion: 0,
-			serverMinorVersion: 0,
-			serverReleaseVersion: 0,
-			engineEditionId: 0,
-			serverVersion: '',
-			serverLevel: '',
-			serverEdition: '',
-			isCloud: false,
-			azureVersion: 0,
-			osVersion: '',
-			options: {}
-		};
-		return info;
-	}
-
-	public get connectionId(): string {
-		return this._connectionId;
-	}
-
-	public get host(): string {
-		if (!this._host) {
-			this.ensureHostAndPort();
-		}
-		return this._host;
-	}
+	private _port: string;
+	private _user: string;
+	private _pass: string;
```

Comment:
`_password`. In our codebase we aim to be clear in naming even if it means a few extra characters.

Reply:
fixed.

Reply:
Can we make one variable name for the same things? I named it `pass` since the spark request parameter field is named `pass`. We currently have `serverName`, `server`, `host`, `port`, `knoxport`, `pass`, `password`....